swift/ql/lib/codeql/swift/elements/expr/ApplyExpr.qll

```diff
@@ -45,3 +37,25 @@ class MethodApplyExpr extends ApplyExpr {
 
   override Expr getQualifier() { result = method.getBase() }
 }
+
+private class PartialDotSyntaxBaseIgnoredApplyExpr extends ApplyExpr {
+  private DotSyntaxBaseIgnoredExpr expr;
+
+  PartialDotSyntaxBaseIgnoredApplyExpr() { expr = this.getFunction() }
+
+  override AutoClosureExpr getStaticTarget() { result = expr.getSubExpr() }
+
+  override Expr getQualifier() { result = expr.getQualifier() }
+
+  override string toString() { result = "call to " + expr }
```

Comment:
We can, but I thought this ("call to Foo.instanceMethod", "call to call to Foo.instanceMethod") would more informative than the getStaticTarget-based default ("call to {...}", "call to {...}" resp.), since in both cases the static target is an AutoClosureExpr.

Reply:
Ah, I see. I'm not a fan of that `"call to call to ..."` string, though 😢. I think I'd actually prefer `"call to {...}"` since that's telling the user that this is a call to a autoclosure expression.

Reply:
Yes, I've removed the "call to call to ..." case (`FullDotSyntaxBaseIgnoredApplyExpr.toString`).

Reply:
Oh, awesome. Sorry, I missed that completely 👍. Thanks a lot!